File: ProgrammersGuide/WorkingWithProjects/CurrencyProperties/CurrencyPropertiesExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammersGuide.WorkingWithProjects.CurrencyProperties
{
    [TestClass, TestFixture]
    public class CurrencyPropertiesExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void ReadingCurrencyProperties()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CurrencyProperties/ReadingCurrencyProperties");

            ReadingCurrencyPropertiesExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void SetCurrencyProperties()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CurrencyProperties/SetCurrencyProperties");

            SetCurrencyPropertiesExample.Program.Main();
        }

    }
}

File: ProgrammersGuide/WorkingWithProjects/CurrencyProperties/CurrencyPropertiesExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammersGuide.WorkingWithProjects.CurrencyProperties
	<TestClass, TestFixture> _
	Public Class CurrencyPropertiesExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub ReadingCurrencyProperties()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CurrencyProperties/ReadingCurrencyProperties")
	
	            ReadingCurrencyPropertiesExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub SetCurrencyProperties()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CurrencyProperties/SetCurrencyProperties")
	
	            SetCurrencyPropertiesExample.Program.Main()
        End Sub

	End Class
End Namespace